### ISSUE TITLE
citation property - types should be CreativeWork and Text rather than CreativeWork and URL

### ISSUE THREAD

**Issue by agbeltran** [codemeta/codemeta#268]
The descriptions of the properties in codemeta (https://github.com/codemeta/codemeta/blob/master/properties_description.csv) lists the property [citation](https://schema.org/citation) with types CreativeWork and URL, while in schema schema.org these are CreativeWork and Text 

Should this be fixed to match the schema.org definition?

**Comment by moranegg** [comment]
Yes. Can you propose a PR for this very small change in: https://github.com/codemeta/codemeta/blob/master/properties_description.csv

**Comment by rebeccaringuette** [comment]
related to [https://github.com/codemeta/codemeta/issues/381](https://github.com/codemeta/codemeta/issues/381)
